### ISSUE TITLE
Account recovery request

### ISSUE THREAD

**Issue by cthart** [pypi/support#4236]
### PyPI Username

`cthart`

### Reason for request

Lost 2FA authentication app

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

### Acknowledgement

- [X] I understand that it may take a significant amount of time to process my account recovery request.

**Comment by Thespi-Brain** [comment]
Hi @cthart,

I have taken a look at your account, and in order to start the account recovery process to reset your 2FA, a public repo to your project needs to be linked to your account. Unfortunately, I didn't find any link to a public repo. Without the repo, there is a 7 day wait period from your written intent to reset your account before we can continue with account recovery. Let us know how you would like to proceed. Thanks!

**Comment by cthart** [comment]
What do you need from me? I have been able to trigger uploads to PyPI through a workflow someone else configured on https://github.com/MadSpindel/swish-python/ColinSent from my iPhoneOn 5 Aug 2024, at 22:32, thespibrain ***@***.***> wrote:﻿
Hi @cthart,
I have taken a look at your account, and in order to start the account recovery process to reset your 2FA, a public repo to your project needs to be linked to your account. Unfortunately, I didn't find any link to a public repo. Without the repo, there is a 7 day wait period from your written intent to reset your account before we can continue with account recovery. Let us know how you would like to proceed. Thanks!

**Comment by cthart** [comment]
Can we move this forward?

**Comment by cthart** [comment]
What's next?

**Comment by Thespi-Brain** [comment]
Hello @cthart,

We haven't received any email from you confirming your intent to reset your account. In order to start the account recovery process to reset your 2FA and recovery codes, your account needs to have a valid/active link to a public repo of your project. 

Since there wasn't any active links, as a workaround please send an email to support@pypi.org confirming your intent to reset your 2FA and recovery codes with a link to this issue (https://github.com/pypi/support/issues/4236). There will be a 7 day wait period from that email as part of our verification/security policy, after which we will go ahead and reset your account and go from there.

**Comment by Thespi-Brain** [comment]
Hi @cthart,

I am following up regarding your request. Are you still having an issue regarding your account's email address? Let us know! I will keep this issue open for another week unless I hear from you earlier. After that, if you are still having trouble, feel free to create a new request. Thanks!

**Comment by cthart** [comment]
Yes, I still want the 2FA reset. I have now also emailed support@pypi.org as I missed that email.

**Comment by Thespi-Brain** [comment]
Hi @cthart,

Your 2FA has been reset and a password reset has been issued. You should be able to regain access to your account after completing the password reset. Let us know if you have any more issues. Thanks.

**Comment by cthart** [comment]
Fantastic, thanks!
